### PR TITLE
docs: update refinery docs

### PR DESCRIPTION
**Pull request by fchikwekwe**
## Which problem is this PR solving?

- Update docs in prep for release

### CODE REVIEW

File: refinery_rules.md [hunk 1]
```diff
@@ -499,6 +499,11 @@ The field to check.
 This can name any field in the trace.
 If the field is not present, then the condition will not match.
 The comparison is case-sensitive.
+The field can also include a prefix that changes the span used for evaluation of the field.
+The only prefix currently supported is `root`, as in `root.http.status`.
+Specifying `root.`  causes the condition to be evaluated against the root span.
```
Comment (mjingle) [inline refinery_rules.md:504]: Remove extra space here.
```suggestion
Specifying `root.` causes the condition to be evaluated against the root span.
```

File: refinery_rules.md [hunk 2]
```diff
@@ -499,6 +499,11 @@ The field to check.
...
+For example, if the `Field` is `root.url`, then the condition will be processed using the url  field from the root span.
```
Comment (mjingle) [inline refinery_rules.md:505]: Remove extra space here.
```suggestion
For example, if the `Field` is `root.url`, then the condition will be processed using the url field from the root span.
```

File: refinery_rules.md [hunk 3]
```diff
@@ -499,6 +499,11 @@ The field to check.
...
+The setting `Scope: span` for a rule does not change the meaning of this prefix -- the condition is still evaluated on the root span and is treated as if it were part of the   span being processed.
```
Comment (mjingle) [inline refinery_rules.md:506]: Remove extra space here. 
```suggestion
The setting `Scope: span` for a rule does not change the meaning of this prefix -- the condition is still evaluated on the root span and is treated as if it were part of the  span being processed.
```

File: rules.md [hunk 4]
```diff
@@ -538,6 +538,11 @@ The field to check.
 This can name any field in the trace.
 If the field is not present, then the condition will not match.
 The comparison is case-sensitive.
+The field can also include a prefix that changes the span used for evaluation of the field.
+The only prefix currently supported is `root`, as in `root.http.status`.
+Specifying `root.`  causes the condition to be evaluated against the root span.
+For example, if the `Field` is `root.url`, then the condition will be processed using the url  field from the root span.
+The setting `Scope: span` for a rule does not change the meaning of this prefix -- the condition is still evaluated on the root span and is treated as if it were part of the   span being processed.
```
[the remark this answers is not in the document]
Comment (mjingle) [reply rules.md:545]: @fchikwekwe Do we have to make this spacing fix at the root file where this content is generated from?

[the remark this answers is not in the document]
Comment (fchikwekwe) [reply rules.md:545]: @mjingle good call. I'll update the RulesMeta.yml file with this change.

**Comment by fchikwekwe** [comment]
@mjingle I think it should be up to date now.

**Review by mjingle — APPROVED** [review]
Looks great to me! Thank you so much @fchikwekwe!

**Review by TylerHelmuth — APPROVED** [review]

**Pull request merged**